src/appleseed/renderer/modeling/shadergroup/shaderquery.cpp

```diff
@@ -0,0 +1,241 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2014-2015 Esteban Tovagliari, The appleseedhq Organization
```

Comment:
Should be

    Copyright (c) 2015 Esteban Tovagliari, The appleseedhq Organization

(same change in header file)